<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>809</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Privacy Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="4,1">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="viewportUpdateMode">
       <enum>QGraphicsView::MinimalViewportUpdate</enum>
      </property>
      <property name="rubberBandSelectionMode">
       <enum>Qt::IntersectsItemBoundingRect</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="btnPrev">
          <property name="text">
           <string>&lt;&lt; Previous</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnNext">
          <property name="text">
           <string>Next &gt;&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnReload">
          <property name="text">
           <string>Reload</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListView" name="listView">
        <property name="viewMode">
         <enum>QListView::ListMode</enum>
        </property>
        <property name="uniformItemSizes">
         <bool>true</bool>
        </property>
        <property name="selectionRectVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>809</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen_image">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_folder"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionFit_image_to_screen"/>
   </widget>
   <addaction name="menuOpen_image"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFit_image_to_screen"/>
   <addaction name="actionPrint_Scene_Info"/>
   <addaction name="actionFinish_selection"/>
   <addaction name="actionClearh_selection"/>
  </widget>
  <action name="actionOpen_folder">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/document-open.svg</normaloff>:/icons/icons/document-open.svg</iconset>
   </property>
   <property name="text">
    <string>Open folder...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/system-log-out.svg</normaloff>:/icons/icons/system-log-out.svg</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionFit_image_to_screen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/view-fullscreen.svg</normaloff>:/icons/icons/view-fullscreen.svg</iconset>
   </property>
   <property name="text">
    <string>Fit image to screen</string>
   </property>
  </action>
  <action name="actionPrint_Scene_Info">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/emblem-important.svg</normaloff>:/icons/icons/emblem-important.svg</iconset>
   </property>
   <property name="text">
    <string>Print Scene Info</string>
   </property>
  </action>
  <action name="actionFinish_selection">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/process-stop.svg</normaloff>:/icons/icons/process-stop.svg</iconset>
   </property>
   <property name="text">
    <string>Finish selection</string>
   </property>
  </action>
  <action name="actionClearh_selection">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/icons/icons/edit-clear.svg</normaloff>:/icons/icons/edit-clear.svg</iconset>
   </property>
   <property name="text">
    <string>Clearh selection</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
